app/views/teams/_form.html.slim

```diff
@@ -69,8 +69,8 @@
               = s.link_to_remove 'Remove', class: 'btn btn-default'
       .form-group.form-btn-group
         = f.link_to_add 'Add another source', :sources, class: 'btn btn-primary col-sm-offset-3'
-  -if can?(:update_conference_preferences, @team)
-  = render 'conference_preference_fields', f: f
+  - if can?(:update_conference_preferences, @team) && @team.accepted?
```

Comment:
Something like this? 
`
    can :update_conference_preferences, Team do |team|
      (user.admin? || (user.student? && team.students.include?(user))) && team.accepted?
    end
`